client/src/js/base/MarkdownNotes.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+import Marked from "marked";
+import { Box } from "./Box";
+
+const NotesBox = styled(Box)`
+    overflow-y: scroll;
+    max-height: 375px;
+    margin-bottom: 0px;
+`;
+
+export const MarkdownNotes = ({ notes }) => {
+    const markedNotes = Marked(notes ? notes : "Edit the sample to add additional notes.");
+    return (
+        <NotesBox>
+            <div dangerouslySetInnerHTML={{ __html: markedNotes }} />
+        </NotesBox>
+    );
+};
```

Comment:
- Naming conventions for `styled-components` says that `NotesBox` should be `StyledMarkdownNotes`
- Make base component generic by changing `MarkdownNotes` to `Markdown` and changing prop name to `markdown`
- I don't think the "Edit the sample to..." message looks very good; Just render the `NoneFound` base  component if `notes` is falsey